.github/workflows/release.yml

```diff
@@ -0,0 +1,236 @@
+name: Release public Lambda layer
+
+on:
+  workflow_dispatch:
+    inputs:
+      layerKind:
+        description: 'Layer kind such as: java-wrapper, java-agent, python3, nodejs'
+        required: true
+      sourceLayerARN:
+        description: 'Clone layer from ARN: (find layer ARN from soaking workflow log)'
+        required: true
+      toLayerName:
+        description: 'Publish to layer name (ie: aws-otel_python38_ver-1-1-0)'
+        required: true
+      aws_region:
+        description: 'Deploy to aws region'
```

Comment:
I figured we can use aws cli to get all the regions accessible by current credentials credentials. Something like

Job 1:
`echo "::set-output name=regions::$(aws ec2 describe-regions | jq '[.Regions[].RegionName]')"`

```
Job 2:
matrix:
  region: ${{ fromJson(job1.outputs.regions) }}

```

Reply:
It does not work because aws cli returns ["region1", "region2"], but fromJson() needs [\"region1\", \"region\2"]